blog-homepage/src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import missedArticles from './missed-articles.json'
+import yourArticles from './your-articles.json'
+import Article from './Article'
+import './Articles.css'
+
+//can use same article component by providing different CSS classes
+const articlesArray = missedArticles
+console.log(missedArticles)
+const yourArticlesArray = yourArticles
+console.log(yourArticles)
+
+function App() {
+  return (
+    <div>
+    <h1>For you</h1>
+    <div>
+    <Article articles={yourArticlesArray} articlesType="yourClass"/>
```

Comment:
I'd argue that you should call this component `Articles` as it refers to a plural of articles. Just be being nitpicky